bokeh/models/markers.py

```diff
@@ -288,6 +288,24 @@ class X(Marker):
 # Dev API
 #-----------------------------------------------------------------------------
 
+marker_types = {
+    "asterisk": Asterisk,
+    "circle": Circle,
+    "circle_cross": CircleCross,
+    "circle_x": CircleX,
+    "cross": Cross,
+    "dash": Dash,
+    "diamond": Diamond,
+    "diamond_cross": DiamondCross,
+    "hex": Hex,
+    "inverted_triangle": InvertedTriangle,
+    "square": Square,
+    "square_cross": SquareCross,
+    "square_x": SquareX,
+    "triangle": Triangle,
+    "x": X
+}
+
```

Comment:
can you add a test to maintain the contents of this dict?

Reply:
Done